RIP-410 Remove course navigation placements for add_user, export_grad…

mlincoln opened the pull request:
…e, manage_official_user, provision_user

https://jira-secure.berkeley.edu/browse/RIP-410

mlincoln committed:
RIP-410 Remove course navigation placements for add_user, export_grade, manage_official_user, provision_user

lyttam commented:
Looks good, but without looking at the details I bet a unit test needs updating.

mlincoln commented:
Yeah, the tests were expecting placement and visibility settings that I changed.
I updated the tests, but for some reason cannot run them locally. Maybe I'll get lucky...
